src/components/dropdown/Dropdown.vue

```diff
@@ -62,7 +62,13 @@
              */
             value(value) {
                 this.selectOption(value)
-            }
+            },
+            /**
+             * Emit event when isActive value is changed
+             */
+             isActive(value) {
+               this.$emit('isActiveChanged', value)
```

Comment:
Change to 'active-change' to be more consistent with the others events and done 👍